repos/fdbt-site/src/pages/products/exports.tsx

```diff
@@ -60,39 +81,59 @@ const Exports = ({ csrf, myFaresEnabled, exportEnabled, operatorHasProducts }: G
+                                <thead className="govuk-table__head">
+                                    <tr className="govuk-table__row">
+                                        <th scope="col" className="govuk-table__header">
+                                            Export name
+                                        </th>
+                                        <th scope="col" className="govuk-table__header">
+                                            Export status
+                                        </th>
+                                        <th scope="col" className="govuk-table__header"></th>
                                     </tr>
-                                ))}
-                            </tbody>
-                        </table>
+                                </thead>
+
+                                <tbody className="govuk-table__body">
+                                    {exports?.map((exportDetails) => (
+                                        <tr className="govuk-table__row" key={exportDetails.name}>
+                                            <td className="govuk-table__cell">{exportDetails.name}</td>
+                                            <td className="govuk-table__cell">
+                                                {exportDetails.netexCount === exportDetails.matchingDataCount ? (
+                                                    <strong className="govuk-tag govuk-tag--green">{`EXPORT COMPLETE ${exportDetails.netexCount} / ${exportDetails.matchingDataCount}`}</strong>
+                                                ) : exportDetails.netexCount === 0 ? (
+                                                    <strong className="govuk-tag govuk-tag--blue">{'LOADING'}</strong>
+                                                ) : (
+                                                    <strong className="govuk-tag govuk-tag--blue">{`IN PROGRESS ${exportDetails.netexCount} / ${exportDetails.matchingDataCount}`}</strong>
+                                                )}
+                                            </td>
+                                            <td className="govuk-table__cell">
+                                                {exportDetails.signedUrl ? (
+                                                    <a href={exportDetails.signedUrl}>Download file</a>
+                                                ) : null}
+                                            </td>
+                                        </tr>
+                                    ))}
+                                </tbody>
+                            </table>
+                        )}
+
+                        {showPopup && (
+                            <InfoPopup
+                                title="We are preparing your export"
+                                text="Your export will take a few seconds to show in the table below."
```

Comment:
Minor: is the table below or is it hidden at this point?

Reply:
Yep - checked this with Dominic.